reference/5.1/Microsoft.PowerShell.Core/Add-History.md

```diff
@@ -21,80 +21,109 @@ Add-History [[-InputObject] <PSObject[]>] [-Passthru] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The **Add-History** cmdlet adds entries to the end of the session history, that is, the list of commands entered during the current session.
 
-You can use the Get-History cmdlet to get the commands and pass them to **Add-History**, or you can export the commands to a CSV or XML file, then import the commands, and pass the imported file to **Add-History**.
-You can use this cmdlet to add specific commands to the history or to create a single history file that includes commands from more than one session.
+The `Add-History` cmdlet adds entries to the end of the session history, that is, the list of
+commands entered during the current session.
+
+You can use the `Get-History` cmdlet to get the commands and pass them to `Add-History`, or you can
+export the commands to a CSV or XML file, then import the commands, and pass the imported file to
+`Add-History`. You can use this cmdlet to add specific commands to the history or to create a single
+history file that includes commands from more than one session.
+
+The session history is a list of the commands entered during the session. The session history
```

Comment:
I would move this section above the one above it. It flows better informationally to me. 
**Current:**
- Brief overview of the command itself.
- Informs the user of a way to keep history outside of just the current session with current cmdlet.
- Fills in the user on the necessary information about session histories and how they are built.
- Informs user about **PSReadLine**. 

**Suggested:**
- Brief overview of the command itself.
- Fills in the user on the necessary information about session histories and how they are built.
- Informs the user of a way to keep history outside of just the current session with current cmdlet.
- Informs user about **PSReadLine**.

Reply:
Reordered